book/content/credit/credit.md

```diff
@@ -79,6 +79,29 @@ It's perfectly possible to cite a dataset or software package directly, and most
 [JORS]: https://openresearchsoftware.metajnl.com/
 [JOSS]: https://joss.theoj.org/
 
+## Making sure YOU get cited by using ORCID to increase your visibility
+
+### What is ORCID?
+- ORCID is short for ‘Open Researcher and Contributor iD'
+- ORCID is a long lasting unique identifier for you as a person, comparible to the personal identification number that your government issues to you.
+
+You can watch this short video instead https://vimeo.com/97150912
+
+### Why should you get an ORCID?
+- to distinguish yourself from others with the same or a similar name;
+- to enable others to find all your outputs/related outputs so they can use and cite them
+- have all your outputs related to you, no matter if you change your name or your instute: your outputs stay attached to you! This way, you only have to enter the information once.
+- to access or sign up to services, e.g. publisher requirements, funding management portals (ResearchFish), the CRIS (current research information system) system at your institute, the Zenodo repository, etc.
+- you can add your ORCID to your CV so that anyone can have a look at all your research outputs. 
```

Comment:
Just for international audiences! :)
```suggestion
- you can add your ORCID to your CV/resume so that anyone can have a look at all your research outputs. 
```